reo/nested_outputs.py

```diff
@@ -175,15 +235,27 @@ def list_of_string(inpt):
                   "description": "Net replacement costs for all technologies, in future value, excluding incentives.",
                   "units": "$"
                 },
+                "total_opex_costs_us_dollars": {
```

Comment:
@Bill-Becker should opex be om for consistency?

Reply:
yeah, probably should be consistent. There's a few instances of "opex" now and "om" in the Financial results.

Reply:
Addressed in https://github.com/NREL/REopt_Lite_API/pull/175/commits/2c8c19fb0dfdb0b4126e5fa6e9c21eba20aae0aa; refactored all "opex" to "om".